ran-core/src/main/java/io/ran/MappingClassWriter.java

```diff
@@ -375,109 +393,123 @@ private void createHydrator() {
 
 
 				if (field.getAnnotation(Serialized.class) != null) {
-					ce.load(3);
-					ce.load(0);
+					ce.load("clazz");
+					ce.load("this");
 					ce.getField(getSelf(), "_serializerFor" + field.getName(), Clazz.of(ISerializer.class));
 					{
```

Comment:
Funny use of curly bracers - maybe a java comment to explain what each block does - would help readability (and allow future modifications more easily)